components/formats-bsd/src/loci/formats/Memoizer.java

```diff
@@ -578,9 +576,10 @@ public boolean versionMismatch() throws IOException {
       }
 
       // REVISION NUMBER
-      if (!versionChecking && !FormatTools.VCS_REVISION.equals(revision)) {
-        LOGGER.info("Different Git version: {} not {}",
-          revision, FormatTools.VCS_REVISION);
+      if (!versionChecking &&
+          FormatTools.VERSION.endsWith("-SNAPSHOT")) {
```

Comment:
I don't think we're currently checking for this elsewhere, I can just imagine that it might be useful going forward, especially for external API consumers.